js/repeater-thumbnail.js

```diff
@@ -36,7 +36,10 @@
 		$.fn.repeater.Constructor.prototype.thumbnail_getSelectedItems = function () {
 			var selected = [];
 			this.$canvas.find('.repeater-thumbnail-cont .selectable.selected').each(function () {
-				selected.push($(this));
```

Comment:
leave this line as-is, since data can be appended to the thumbnail item and accessed in that way